explorer/src/components/account/UpgradeableLoaderAccountSection.tsx

```diff
@@ -77,6 +78,10 @@ export function UpgradeableProgramSection({
       <div className="card-header">
         <h3 className="card-header-title mb-0 d-flex align-items-center">
           Program Account
+          <VerifiedBadge
```

Comment:
The badge looks nice because it stands out but I would prefer if you moved into the table card below in a row which has a label "Anchor Verification Status"